### DISCUSSION TITLE
Lapor domain yang mengandung virus/malware reportnya kemana?

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1341] [Q&A]
https://www.wordpressthemesall.com/maktub-nulled-v-1-4-1-minimal-lightweight-blog-for-wordpress-free-download/

example link: (contain virus)
https://www.wordpressthemesall.com/download/Maktub1.4.1v.zip

code inject virus:
https://www.wordpressthemesall.com/wz.txt

saya cek dari virustotal menunjukkan clean, ini karena kodenya obfuscate

**Comment by realodix** [comment]
Saya kurang tahu deh harus lapor kemana?

Setahu Saya bisa lapor ke:
- https://www.phishtank.com/
- https://openphish.com/
- https://urlhaus.abuse.ch/

Nanti dari sana akan didistribusikan sebagai oleh https://gitlab.com/malware-filter
